capa/render/result_document.py

```diff
@@ -501,9 +510,23 @@ class MaecMetadata(FrozenModel):
     model_config = ConfigDict(frozen=True, populate_by_name=True)
 
 
+@lru_cache(maxsize=None)
+def load_rules_prevalence() -> Dict[str, str]:
+    CD = Path(__file__).resolve().parent.parent.parent
+    file = CD / "assets" / "rules_prevalence_data" / "rules_prevalence.json.gz"
+    if not file.exists():
+        raise FileNotFoundError(f"File '{file}' not found.")
```

Comment:
we may want to return the empty dict, rather than raising a fatal error here. is this file strictly required in order to instantiate any capa rule?